lib/neo4j/shared/attributes.rb

```diff
@@ -0,0 +1,207 @@
+module Neo4j::Shared
```

Comment:
It would be good to have comments at the top with SOME CAPS to indicate where this file originally came from so that people can understand the history